File: 251229/상품 코드/product-code.js
const fs = require('fs');

class Product {
    constructor(name, code) {
        this.name = name;
        this.code = code;
    }
}

const [name, code] = fs.readFileSync(0).toString().split(' ');

const product1 = new Product('codetree', 50);
const product2 = new Product(name, Number(code));

console.log(`product ${product1.code} is ${product1.name}`);
console.log(`product ${product2.code} is ${product2.name}`);

File: 251229/상품 코드/product-code.py
class Product:
    def __init__(self, name, code):
        self.name = name
        self.code = code


name, code = input().split()

product1 = Product('codetree', 50)
product2 = Product(name, int(code))

print(f'product {product1.code} is {product1.name}')
print(f'product {product2.code} is {product2.name}')